com.trollworks.gcs/src/com/trollworks/gcs/weapon/WeaponDamage.java

```diff
@@ -237,6 +248,18 @@ public void setType(String type) {
         }
     }
 
+    public Integer getPercentBonus() {
+        return mPercentBonus;
+    }
+
+    public void setPercentBonus(Integer percentBonus) {
+        if (mPercentBonus != percentBonus) {
+            mPercentBonus = percentBonus;
+            notifyOfChange();
+        }
+
```

Comment:
Please remove dead space like this at the beginning or end of functions (I've noticed it in a few other spots, too).